x-pack/qa/sql/src/main/resources/string-functions.sql-spec

```diff
@@ -0,0 +1,80 @@
+stringAscii
+// tag::ascii
```

Comment:
the `tag/end` is used in the docs - not sure whether this is the case here so they should be removed (since it's just noise).